avalon/pipeline.py

```diff
@@ -1325,31 +1333,36 @@ def path_from_config():
             "representation": representation["name"],
             "user": Session.get("AVALON_USER", getpass.getuser()),
             "app": Session.get("AVALON_APP", ""),
-            "task": Session.get("AVALON_TASK", "")
+            "task": Session.get("AVALON_TASK", ""),
+            "hierarchy": asset["data"].get("hierarchy", "")
         }
 
         try:
             path = template.format(**data)
         except KeyError as e:
-            log.debug("Template references unavailable data: %s" % e)
+            log.warning("Missing value for filepath template: {}".format(e))
```

Comment:
Could be made *even* more clear I think. How about this?

```
"One of the variables used in this filepath is unavailable: {}, these are the variables available: {}".format(
  path, ", ".join(data.keys()))
```

Reply:
Changing all these logs to warnings I think will get confusing quick for most artists. Say your pipeline relies not on the first behavior but the second or third. In that case every time you are trying to get a representation the artist will end up seeing the first two warnings. E.g. when loading assets.

Having better messaging I think is good. I am just wondering whether making these warnings is the most sensible direction.

Reply:
Yeah, better to keep the log level at `debug` I think.

Reply:
Agreed.

Reply:
Lowered.